analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -35,4 +71,192 @@ class Bundler : PackageManager() {
+                            version = version
+                    ),
+                    declaredLicenses = declaredLicenses.toSortedSet(),
+                    aliases = emptyList(),
+                    vcs = vcsDir?.getInfo(workingDir) ?: VcsInfo.EMPTY,
+                    homepageUrl = projectHomepageUrl,
+                    scopes = scopes.toSortedSet()
+            )
+            return AnalyzerResult(
+                    true,
+                    project,
+                    packages.map { it.toCuratedPackage() }.toSortedSet(),
+                    errors
+            )
+        } finally {
+            // Delete vendor folder to not pollute the scan.
+            if (!vendorDir.deleteRecursively()) {
+                throw IOException("Unable to delete the '$vendorDir' directory.")
+            }
+
+            // Restore any previously existing "vendor" directory.
+            if (tempVendorDir != null) {
+                log.info { "Restoring original '$vendorDir' directory from '$tempVendorDir'." }
+                Files.move(tempVendorDir.toPath(), vendorDir.toPath(), StandardCopyOption.ATOMIC_MOVE)
+                if (!tempVendorDir.parentFile.delete()) {
+                    throw IOException("Unable to delete the '${tempVendorDir.parent}' directory.")
+                }
+            }
+        }
+    }
+
+    private fun parseScope(workingDir: File, groupName: String, dependencyList: List<String>, scopes: MutableSet<Scope>,
+                           packages: MutableSet<Package>, errors: MutableList<String>) {
+        log.debug("Parsing scope: $groupName\nscope top level deps list=$dependencyList")
+        val scopeDependencies = mutableSetOf<PackageReference>()
+        dependencyList.forEach {
+            parseDependency(workingDir, it, packages, scopeDependencies, errors)
+        }
+        scopes.add(Scope(groupName, true, scopeDependencies.toSortedSet()))
+    }
+
+    private fun parseDependency(workingDir: File, gemName: String, packages: MutableSet<Package>,
+                                scopeDependencies: MutableSet<PackageReference>, errors: MutableList<String>) {
+        log.debug("Parsing dependency: $gemName")
+        try {
+
+            val (_, version, homepageUrl, declaredLicenses, description, dependencies) = getGemDetails(gemName,
+                    workingDir)
+            packages.add(Package(
+                    id = Identifier(
+                            provider = javaClass.simpleName,
+                            namespace = "",
+                            name = gemName,
+                            version = version
+                    ),
+                    declaredLicenses = declaredLicenses,
+                    description = description,
+                    homepageUrl = homepageUrl,
+                    binaryArtifact = RemoteArtifact.EMPTY,
+                    sourceArtifact = RemoteArtifact.EMPTY,
+                    vcs = parseVcs(homepageUrl))
+            )
```

Comment:
I believe `vcsProcessed` should also get assigned here via `processPackageVcs(...)`.

Reply:
Yes, like this:

```
val vcsFromPackage = parseVcs(homepageUrl)
...
  vcs = vcsFromPackage,
  vcsProcessed = processPackageVcs(vcsFromPackage)
```